### PR TITLE
Add RPM black box field

### PR DESCRIPTION
**Pull request by tbolin**
Adds a field for RPM as generated by https://github.com/betaflight/betaflight/pull/12562.

Still WIP, but basic functionality is there.

### Todo
- [x] Add RPM field to the list in Graph setup -> Add graph (currently Custom graph has to be used) 
- [x] Fix plot scaling so all RPM plots are use the same scale.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: js/flightlog.js
Comment (haslinghuis) [inline js/flightlog.js:580]:
```suggestion
             fieldNameToIndex["eInterval[4]"], fieldNameToIndex["eInterval[5]"], fieldNameToIndex["eInterval[6]"], fieldNameToIndex["eInterval[7]"]];
```

**Review by haslinghuis — DISMISSED** [review]

**Comment by nerdCopter** [comment]
@tbolin , PR needs conflict resolutions

**Comment by haslinghuis** [comment]
No worries - we won't merge until firmware PR is done.

**Comment by tbolin** [comment]
Closing this PR in favor of the (e)RPM one.

**Pull request closed without merging**
